source/docs/guides/3D_secure.md

```diff
@@ -52,40 +52,53 @@ curl https://api.na.bambora.com/v1/payments \
 -H "Authorization: Passcode MzAwMjAwNTc4OjRCYUQ4MkQ5MTk3YjRjYzRiNzBhMjIxOTExZUU5Zjcw" \
 -H "Content-Type: application/json" \
 -d '{
-    "payment_method":"token",
-    "order_number":"MyOrderId-01234",
-    "amount":15.99,
-    "token":{
-        "code":"gt7-0f2f20dd-777e-487e-b688-940b526172cd",
-        "name":"John Doe",
-		"3d_secure":{
-			"enabled":true
-		}
-    },
-    "term_url": "https://my.merchantserver.com/redirect/3d-secure"
+
+   "amount": 250.01,
+   "payment_method":"token",
+   "customer_ip": "123.123.123.123"
+   "token": {
+      "name": "Test User",
+      "code":"gt7-0f2f20dd-777e-487e-b688-940b526172cd",
+      "3d_secure": {
+         "browser": {
+            "acceptHeader": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3",
```

Comment:
These need to be updated with the underscore as well

Reply:
doh! fixed now.